src/Analysis/Codelyzer.Analysis.Build/WorkspaceBuilderHelper.cs

```diff
@@ -536,20 +533,19 @@ private OSPlatform DetermineOSPlatform()
         private AnalyzerManager GetAnalyzerManager()
         {
             AnalyzerManager analyzerManager;
+            var analyzerManagerOptions = new AnalyzerManagerOptions
+            {
+                LogWriter = _writer
+            };
+
             if (IsSolutionFile())
```

Comment:
Is it possible to add a test that would have failed without this change?

Reply:
This was a refactor to reduce code duplication so original tests should suffice for this change